Frontend/ui-customer/src/Resources/Localizable/Language/HI.purs

```diff
@@ -362,4 +362,8 @@ getHI script = case script of
   REFERRAL_CODE_APPLIED -> "रेफ़रल लागू!"
   HEY -> "नमस्ते"
   INVALID_CODE_PLEASE_RE_ENTER -> "अवैध कोड। कृपया पुनः दर्ज करें"
-  LET_TRY_THAT_AGAIN ->  "आइए फिर से कोशिश करें..."
\ No newline at end of file
+  LET_TRY_THAT_AGAIN ->  "आइए फिर से कोशिश करें..."
+  CHOOSE_YOUR_RIDE -> "अपनी सवारी चुनें"
+  END_RIDE -> "End Ride"
```

Comment:
Can you add the translation for end ride as well

Reply:
done